app/src/main/java/org/fossasia/openevent/app/core/attendee/list/AttendeesFragment.java

```diff
@@ -161,13 +165,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_attendees, container, false);
-        binding.fabScanQr.setOnClickListener(v -> {
-            Intent scanQr = new Intent(context, ScanQRActivity.class);
-            scanQr.putExtra(MainActivity.EVENT_KEY, eventId);
-            startActivity(scanQr);
-        });
-
         binding.fabScanQr.getDrawable().setColorFilter(getResources().getColor(android.R.color.white), PorterDuff.Mode.SRC_ATOP);
+
+        if (PLAYSTORE.equals(BuildConfig.FLAVOR)) {
+            binding.fabScanQr.setOnClickListener(v -> scanningDecider.openScanQRActivity(getActivity(), eventId));
+        } else {
+            binding.fabScanQr.setOnClickListener(v -> Toast.makeText(context, R.string.scanning_disabled_message, Toast.LENGTH_SHORT).show());
+        }
```

Comment:
Unconditionally call the method on scanningDecider, and in fdroid version of the method, show the toast